railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_5_2.rb.tt

```diff
@@ -13,3 +13,6 @@
 # Use AES 256 GCM authenticated encryption for encrypted cookies.
 # Existing cookies will be converted on read then written with the new scheme.
 # Rails.application.config.action_dispatch.use_authenticated_cookie_encryption = true
```

Comment:
Ideally, we should be able to remove this config and use the below one to infer it. But that's for another time.

cc @mikeycgto